src/c/sk_bitmap.cpp

```diff
@@ -300,3 +320,9 @@ sk_colortable_t* sk_bitmap_get_colortable(sk_bitmap_t* cbitmap)
 {
     return ToColorTable(AsBitmap(cbitmap)->getColorTable());
 }
+
+void sk_bitmap_set_pixels(sk_bitmap_t* cbitmap, void* pixels, sk_colortable_t* ctable)
+{
+    SkBitmap* bmp AsBitmap(cbitmap);
```

Comment:
I think there is something wrong here. no equals (=) and this is a dupe of the function above...